IanGRap/Squish
# Main menu overhaul

Contributors: Reed, Athene, Austin, Ian

Goal: Edit assets on the title screen to make it look nicer:

Important Changes:
- Lobby.unity: Added and moved art for title screen (including Athene's tree)
- UIController.cs: Added controls to change text color and size when scrolling over options.

How To Test:
-Load up the lobby and look at the Main Menu. Does it look nice. Does any area look like it missing art. Go through the lobby to a match and see if any art seems missing or unnecessarily changed.
